h/static/scripts/widget-controller.js

```diff
@@ -97,8 +117,19 @@ module.exports = function WidgetController(
     visibleThreads.setRootThread(thread());
     $scope.selectedTab = annotationUI.getState().selectedTab;
 
+    if (countWaitingToAnchorAnnotations(annotationUI.getState().annotations) > 0) {
```

Comment:
This can be simplified to `$scope.waitingToAnchorAnnotations = countWaitingToAnchorAnnotations(...) > 0`